src/css/bestsellers.css

```diff
@@ -0,0 +1,113 @@
+.bestsellers {
+  padding: 64px 0;
+}
+.bestsellers-container {
+  display: flex;
+  flex-direction: column;
+}
+.bestsellers-title {
+  margin-bottom: 21px;
+  text-align: center;
+  color: var(--primary-color-blue-font-0a224a);
+  font-size: 28px;
+  font-style: normal;
+  text-transform: uppercase;
+}
+
+.bestsellers-list {
+}
+.bestsellers-item {
+}
+
+.bestsellers-img {
+  width: 100%;
+  height: auto;
+  margin-bottom: 14px;
+}
+.bestsellers-furniture-text {
+  margin-bottom: 10px;
+  color: var(--primary-color-blue-font-0a224a);
+  font-style: normal;
+  line-height: 1.5;
+}
+.bestsellers-price {
+  color: var(--primary-color-blue-font-0a224a);
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 1.5;
+}
+/* ----------------------------------------------- */
+
+/* @media screen and (min-width: 768px) and (max-width: 1157px) {
+} */
+
+/* ------------------------ */
+@media screen and (max-width: 767px) {
+  .bestsellers-item:nth-child(n + 3) {
+    display: none;
+  }
+  .bestsellers-item:not(:last-child) {
+    margin-bottom: 30px;
+  }
+  .bestsellers-main-wrapper {
+    margin-bottom: 30px;
+  }
+  .bestsellers-container {
+    flex-direction: column;
+  }
+}
+
+@media screen and (min-width: 768px) {
+  .bestsellers {
+    padding: 100px 0;
+  }
+  .bestsellers-title {
+    font-size: 32px;
+    line-height: 1.41;
+  }
+  .bestsellers-list {
+    display: flex;
+    justify-content: center;
+    flex-wrap: wrap;
+    gap: 32px;
+  }
+  .bestsellers-item {
+    width: calc((100% - 32px) / 2);
+  }
+  .bestsellers-img {
+    margin-bottom: 15px;
+  }
+  .bestsellers-furniture-text {
+    font-size: 18px;
+    margin-bottom: 7px;
+  }
+  .bestsellers-price {
+    font-size: 18px;
+  }
+}
+
```

Comment:
@media screen and (min-width: 768px) and (max-width: 1439px)
.bestsellers-main-wrapper {
    margin-bottom: 32px;
}